Project-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,7 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.container {
+  width: 500px;
```

Comment:
For some reason when I look at it, it isn't getting on constrained by the width.